QuickPay/Services/PaymentsService.cs

```diff
@@ -121,7 +121,6 @@ public Task<Payment> CancelPayment(int id, string callbackUrl = null, bool synch
             };
 
             var url = "payments/" + id + "/cancel";
-            url += "?synchronized=" + synchronized;
```

Comment:
`?synchronized` do not care about the value, just it's presents.
So `?synchronized` should only be added if `synchronized == true`
ex.
```
if (synchronized) {
  url += "?synchronized";
}
  ```